src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -248,4 +248,17 @@ export class WorkItemListComponent implements OnInit, AfterViewInit {
     this.showTypesOptions = false;
     this.router.navigate(['/work-item-list/detail/new?' + type]);
   }
-}
\ No newline at end of file
+
+  onDragStart() {
+    console.log('on drag start');
+  }
+
+  //Event listener for WI drop.
+  onDragEnd(workItemId: string) {
+    // Build the ID-index map after rearrange
```

Comment:
Mention that re arrange is happening inside drag and drop library in the comment.